SpecFlowSolar/Features/CartTests.feature

```diff
@@ -0,0 +1,11 @@
+﻿Feature: CartTests
+@mytag
+Scenario: add and delete product and return to home
+	Given open home page in CartTests
+	When open solar cabels in menu in CartTests
+	And add to cart one product with index 1
+	And go to cart in modal
+	And remove product from cart if one
```

Comment:
and "if not one" then do not remove? Test case should not contain "if". I understand this is copy/paste from HW11, but even if you have some internal names in Pages and Steps, these feature steps should be clear to people who read this